test/util/helpers.test.js

```diff
@@ -15,12 +15,19 @@ test('createGiftCardRequest', () => {
     expect(result.value.amount).toBe(20)
 })
 
-test('getEndpoint', () => {
-    const endpoint = 'EU'
-    const environment = 'production'
-    const result = getEndpoint(endpoint, endpoints, environment)
-    expect(result.location).toBe('EU')
-    expect(result.environment).toBe('production')
-    expect(result.host).toBe('agcod-v2-eu.amazon.com')
-    expect(result.region).toBe('eu-west-1')
+test('getEndpoints', () => {
+    const endpointNames = ['NA', 'EU', 'FE']
+    const environments = ['sandbox', 'production']
+    const hosts = ['agcod-v2-gamma.amazon.com', 'agcod-v2.amazon.com', 'agcod-v2-eu-gamma.amazon.com', 
+                            'agcod-v2-eu.amazon.com', 'agcod-v2-fe-gamma.amazon.com', 'agcod-v2-fe.amazon.com']
+    const regions = ['us-east-1', 'eu-west-1', 'us-west-2']
+    for (i = 0; i < 3; i++) {
```

Comment:
can you use the array length instead of hardcoding 3?

Reply:
Done